newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardSortOptionsDialog.js

```diff
@@ -0,0 +1,220 @@
+// @flow
+import React from 'react';
+import { t, Trans } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import { type I18n as I18nType } from '@lingui/core';
+
+import Dialog from '../../UI/Dialog';
+import FlatButton from '../../UI/FlatButton';
+import { ColumnStackLayout } from '../../UI/Layout';
+import RaisedButton from '../../UI/RaisedButton';
+import SelectField from '../../UI/SelectField';
+import SelectOption from '../../UI/SelectOption';
+import Text from '../../UI/Text';
+import TextField from '../../UI/TextField';
+
+import { type LeaderboardSortOption } from '../../Utils/GDevelopServices/Play';
+import { Column, LargeSpacer, Line } from '../../UI/Grid';
+import HelpButton from '../../UI/HelpButton';
+import Checkbox from '../../UI/Checkbox';
+import { FormHelperText } from '@material-ui/core';
+import { MarkdownText } from '../../UI/MarkdownText';
+
+const isWholeNumber = (value: any): boolean =>
```

Comment:
Can you reuse the function for leaderboardAppearanceDialog if they have the same name, or do they have a different logic?